src/extensions.rs

```diff
@@ -195,4 +195,15 @@ impl<T: AsRef<str>> T {
         let index = s.find(c)?;
         Some((&s[..index], &s[index + c.len_utf8()..]))
     }
+
+    fn split_at_word<'a>(&'a self, split_at: &str) -> (String, String) {
+        let mut words = self.as_ref().trim().split(' ').collect_vec();
+        match words.iter().position(|w| w == &split_at) {
+            Some(word_ind) => {
+                let right_side = words.split_off(word_ind + 1).join(" ");
```

Comment:
This'll do the thing i mentioned above, where it will not include the word we are splitting by on any of the two sides, which I'd say is how i'd expect the function to behave
```suggestion
                let right_side = words.split_off(word_ind + 1).join(" ");
                words.pop();
```